srcpkgs/cinnamon/template

```diff
@@ -27,7 +25,9 @@ license="GPL-2.0-or-later, LGPL-2.0-or-later"
 homepage="http://developer.linuxmint.com/projects/cinnamon-projects.html/"
 changelog="https://raw.githubusercontent.com/linuxmint/Cinnamon/${version}/debian/changelog"
 distfiles="https://github.com/linuxmint/${pkgname}/archive/${version}.tar.gz"
-checksum=fcc6182a074b15d599dd53014086bb77b0bdceb3e667627e9125331f5ad0195e
+checksum=78b3e9716bbe78a85f723b17c15d6e405c8b7b80b06ebae052b03cb66bafdd71
+patch_args=" -Np1" # upstream fix (originally from archlinux):
+# https://github.com/linuxmint/cinnamon/pull/9725
```

Comment:
This information should go in the patch.

Reply:
Ok, I will add it as a comment, thank you.

Reply:
Done, thank you!